renderdoc/driver/gl/gl_shader_refl.cpp

```diff
@@ -283,7 +283,7 @@ GLuint MakeSeparableShaderProgram(WrappedOpenGL &gl, GLenum type, vector<string>
   gl.glGetProgramiv(sepProg, eGL_LINK_STATUS, &status);
 
   // allow any vertex processing shader to redeclare gl_PerVertex
-  if(status == 0 && type != eGL_FRAGMENT_SHADER && type != eGL_COMPUTE_SHADER)
+  if(!IsGLES && status == 0 && type != eGL_FRAGMENT_SHADER && type != eGL_COMPUTE_SHADER)
```

Comment:
What's the impact of not doing this for GLES? Does GLES not have the stupid rule about having to explicitly declare gl_PerVertex to be made separable?